sam-karis/airtech
# Ft book tickets 167660075

## What does this PR do?
------
It adds tickets booking, view and filter flight tickets by date functionality

## Description of Task to be completed?
-----
- add ticket model, serializers, URLs, and views
- update flight model
- update test

## How should this be manually tested?
------
clone the repo
check out to `ft-book-tickets-167660075`
add environment variables as defined in [`.env_example`](https://github.com/sam-karis/airtech/blob/develop/.env_example)
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[book tickets](https://www.pivotaltracker.com/story/show/167660075)


## Screenshots (if appropriate)
------
#### view all your tickets for a flight
![image](https://user-images.githubusercontent.com/22454909/63476791-0f6fe380-c48b-11e9-90cc-46579c284a39.png)



#### view flight tickets for a specific date
![image](https://user-images.githubusercontent.com/22454909/63476810-20b8f000-c48b-11e9-8ce1-453708d5d1a9.png)


#### Book ticket
![image](https://user-images.githubusercontent.com/22454909/63476725-d20b5600-c48a-11e9-894e-c606b74dfa52.png)
